ckcp/openshift_dev_setup.sh

```diff
@@ -76,8 +71,35 @@ parse_args() {
 }
 
 init() {
-  APP_LIST="openshift-gitops compute ckcp register-compute"
-  cluster_type="openshift"
+  APP_LIST=(
+            "openshift-gitops" 
+            "ckcp"
+           )
+  # get the list of APPS to be installed
+  read -ra APPS <<< "$(yq eval '.APPS | join(" ")' "$CONFIG")"
+  for app in  "${APPS[@]}"
+  do
+    APP_LIST+=("$app")
+  done
+
+  # get cluster type
+  cluster_type=$(yq '.CLUSTER_TYPE' "$CONFIG")
+  cluster_type=${cluster_type:-"openshift"}
+
+  GIT_URL=$(yq '.GIT_URL' "$CONFIG")
+  GIT_REF=$(yq '.GIT_REF' "$CONFIG")
+  # set default values in case they're missing
+  GIT_URL=${GIT_URL:-"https://github.com/openshift-pipelines/pipeline-service.git"}
+  GIT_REF=${GIT_REF:="main"}
```

Comment:
These lines can be merged to get value and provide defaults using `yq` only as Xin has suggested above. See https://mikefarah.gitbook.io/yq/operators/alternative-default-value